5_kata.py

```diff
@@ -0,0 +1,14 @@
+#https://www.codewars.com/kata/58f6000bc0ec6451960000fd/train/python
+
+def sel_reverse(arr,l):
+    if l>0 and len(arr) > l:
+        qty = len(arr)
```

Comment:
Rename the variable to something more meaningful. OR you can also skip assigning `len(arr)` to this variable, and use `len(arr)` in the line below.